lib/Makefile

```diff
@@ -292,16 +295,20 @@ libzstd : $(LIBZSTD)
 
 .PHONY: lib
 lib : libzstd.a libzstd
+	@if [ $(ZSTD_NO_MULTITHREAD) = 1 ]; then \
```

Comment:
are you sure of this syntax ?

Reply:
It seems like this works? I'm not too familiar with what's acceptable in a Makefile, though I suppose another way to emit this message would be to use the Makefile syntax, and do something like:
```
ZSTD_MULTITHREAD_MSG = multithreading disabled
ifneq ($(ZSTD_NO_MULTITHREAD), 1)
  ZSTD_MULTITHREAD_MSG = multithreading enabled
endif

lib : libzstd.a libzstd
	@echo $(ZSTD_MULTITHREAD_MSG)
```

Is this better?

Reply:
Yes, this construction would be a bit more familiar.